scripts/lis_hyperv_platform/lis-rpm-hyperv-tests-lisav2.ps1

```diff
@@ -0,0 +1,96 @@
+[CmdletBinding()]
+param(
+    [parameter(Mandatory = $true)]
+    [String] $JobId,
+    [parameter(Mandatory = $true)]
+    [String] $DistroVersion,
+    [parameter(Mandatory = $true)]
+    [String] $TestCategory,
+    [String] $TestArea,
+    [String] $TestNames,
+    [string] $TestTag,
+    [string] $TestPriority,
+    [String] $LISAImagesShareUrl,
+    [String] $LisUrl,
+    [String] $LisOldUrl,
+    [String] $ExcludeTests,
+    [String] $IncludeTests,
+    [string] $ExecutionTag,
+    [String] $Delay
+)
+
+foreach ($key in $MyInvocation.BoundParameters.keys) {
+    $value = (get-variable $key).Value
+    write-host "$key -> $value"
+}
+
+function Main {
+    if (!$TestCategory) { $TestCategory = "All" }
+    if (!$TestArea)     { $TestArea = "All" }
+    if (!$TestNames)    { $TestNames = "All" }
+    if (!$TestTag)      { $TestTag = "All" }
+    if (!$TestPriority) { $TestPriority = "All" }
+
+    Write-Output "Sleeping $Delay seconds..."
+    Start-Sleep $Delay
+
+    Write-Host "Getting the proper VHD folder name for LISA with $DistroVersion"
+    $imageFolder = Join-Path $LISAImagesShareUrl $DistroVersion.split("_")[0]
+    $imageFolder = Join-Path $imageFolder $DistroVersion
+    $parentVhd = $(Get-ChildItem $imageFolder | Where-Object { $_.Extension -eq ".vhd" -or $_.Extension -eq ".vhdx"} | Sort LastWriteTime | Select -Last 1).Name
+    $VHD_Path = Join-Path $imageFolder $parentVhd
+    $VMgeneration = "1"
+    if ($DistroVersion -like "*gen2vm*") {
```

Comment:
This is one of Gen2 VM name, "SUSE SLES-SAP gen2-12-sp4 2019.05.13" Please make sure of the correct pattern in the condition.

Reply:
This is HyperV Test. No reference to any Azure test here.